utils/docker/docker.go

```diff
@@ -0,0 +1,10 @@
+package dockermock
```

Comment:
Remove this file, you can generate mock directly from vendor

```mockery -name CommonAPIClient -dir vendor/github.com/docker/docker/client -inpkg```

Reply:
I didn't want to access to vendor folder directly. It's a bit messy and we can easily avoid it like with the current implementation.